### ISSUE TITLE
UseeTv

### ISSUE THREAD

**Issue by rollykaunang** [intact/livestreamer#1]
@intact Couple months ago I asked you to make a plugin for useetv due to the site using geo ip it could only watch from indonesia territory so i guess you could not make it. "useetv.com" will block if it detect ip address from outside indonesia. I've tried with expressvpn it work very well, if You don't mind could you make a plugin i will provide you expressvpn along with activation code for one month. Please let me know. Thank You So Much.

**Comment by intact** [comment]
@rollykaunang you can try https://github.com/intact/livestreamer/blob/useetv/src/livestreamer/plugins/useetv.py

**Comment by rollykaunang** [comment]
@intact The remarkable smart and genius guy Thank You so Much!

**Comment by rollykaunang** [comment]
@intact hello, sorry bother you again and again. Please take a look useetv.com they make minor changing on their site i have this error 
C:\Program Files (x86)\Livestreamer>livestreamer http://www.useetv.com/livetv/antv best
[cli][info] Found matching plugin useetv for URL http://www.useetv.com/livetv/antv
error: Unable to open URL: http://118.97.167.8/jhos/antv.m3u8?stime=20170111150000&etime=20170111190000&encoded=0ab5669da880feef52084&ce=fa161aa74311 (404 Client Error: Not Found)

Thank You So Much.

**Comment by intact** [comment]
Looks like a server problem, plugin works for me right now.
```
./livestreamer http://www.useetv.com/livetv/net best
[cli][info] Found matching plugin useetv for URL http://www.useetv.com/livetv/net
[cli][info] Available streams: 180p (worst), 360p, 720p (best)
[cli][info] Opening stream: 720p (hls)
[cli][info] Starting player: ffplay
[cli][info] Player closed
[cli][info] Stream ended
```

**Comment by rollykaunang** [comment]
Sorry for late response.
It seems my windows 10 got updated. It affected livestreamer. Yes it is
work perfect on windows 7 and ubuntu. Thank You So much Intac.

**Comment by unlocKing** [comment]
@rollykaunang Can you check for me if Australia Plus is working through your VPN?

It seems the foreign channels aren't working for me:

'[cli][info] Found matching plugin useetv for URL http://www.useetv.com/livetv/ausplus
[cli][info] Available streams: 360p (worst, best)
[cli][info] Opening stream: 360p (hls)
[cli][error] Could not open stream: Unable to open URL: http://edge.telin.swiftserve.com/live/sejXc8RhId_livetv-sbr/d41d8cd98f/chunklist.m3u8?stime=20170113210000&etime=20170114010000&encoded=01cf11582b9ddbd4a1330& (403 Client Error: Permission denied)'


'[cli][info] Found matching plugin useetv for URL http://www.useetv.com/livetv/ftv
[cli][info] Available streams: 360p (worst, best)
[cli][info] Opening stream: 360p (hls)
[cli][error] Could not open stream: Unable to open URL: http://edge.telin.swiftserve.com/live/sejXc8RhId_livetv-sbr/d41d8cd98f/chunklist.m3u8?stime=20170113210000&etime=20170114010000&encoded=01cf11582b9ddbd4a1330& (403 Client Error: Permission denied)'


Thanks.

**Comment by rollykaunang** [comment]
@intac
Hello Intac, I use torguard vpn, I've tried it. Unfortunately i was not
work i got message similar to yours. Then i've tried using Mozilla firefox
browser and i got message like this* "Alert!* Layanan ini hanya dapat
dinikmati pelanggan INDIEHOME" <http://cable.useetv.com/>
It's mean " Alert! This service for INDIHOME customer only" I asked my
friend. He said the company using setup box for cable customer.

All The Best.

**Comment by intact** [comment]
Yes, some channels are available only for Indiehome subscribers. Maybe i can later add better error message for such channels.

**Comment by unlocKing** [comment]
Thanks, I was wondering why those channels would not play..

Thought it was due to the x-forward I was using which worked on the local channels:

`livestreamer --http-header "X-Forwarded-For=118.82.8.8" http://www.useetv.com/livetv/ausplus best`

**Comment by rollykaunang** [comment]
Yes You are right it is For indiehome subscribers only. Yes please if U
have time to add better message.
Thank You.
